src/components/Settings/SettingsLogs/index.tsx

```diff
@@ -77,19 +77,19 @@ const SettingsLogs: React.FC = () => {
   const { data: appData } = useSWR('/api/v1/status/appdata');
 
   useEffect(() => {
-    const displayString = window.localStorage.getItem('logs-display-settings');
+    const filterString = window.localStorage.getItem('logs-filter-settings');
```

Comment:
I don't mind changing `displayString` to `filterString` but since we already committed to `logs-display-settings` lets just stick to it. Otherwise, people will have that value sitting around in their local storage forever.